andela/bp-esa-frontend
# #167296777 Change Numbering of Automations in List View

#### What does this PR do?
- [x] change numbering of list view automation

#### Description of Task to be completed?

- in the list view of automation, the numbering in the first
the column now reflects the page the table is on

#### How should this be manually tested?
- pull the branch locally start the development server &rarr; `yarn start`
- navigate to `http://localhost:3000/?view=listView`
- navigate between pages and observe the updated numbering system

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#167296777](https://www.pivotaltracker.com/story/show/167296777)

#### Postman Documentation

N/A

#### Screenshots (If applicable)

![image](https://user-images.githubusercontent.com/15332525/61522855-1e3a2500-aa0b-11e9-83ac-91cb4c0ff05a.png)

![image](https://user-images.githubusercontent.com/15332525/61522863-25613300-aa0b-11e9-976d-d65bc26762d6.png)
